src/modules/Artists/selectors/index.js

```diff
@@ -3,3 +3,7 @@ export const selectArtists = state => state.artistReducer.artists;
 export const selectAlbums = artistId => state => state.albumReducer.albums[artistId];
 
 export const selectTracks = albumId => state => state.trackReducer.tracks[albumId];
+
+export const loadingStatus = state => state.loadingReducer.loading;
+
+export const error = state => state.errorReducer.error;
```

Comment:
It would be better to create a separate module for notification and create a NotificationWrapper component. We could place redux logic here as well